interface/src/app/tribe_client/tribe_client.js

```diff
@@ -1,42 +1,45 @@
 angular.module('adage.tribe_client', [
-    'adage.tribe_client.resource',
-    'adage.tribe_client.directives',
+  'adage.tribe_client.resource',
+  'adage.tribe_client.directives',
+  'adage.tribe_client.genesets'
 ])
 
-.factory( 'UserFactory', ['User', function( User ) {
-    var promise = null;
-    var user = null;
+.factory('UserFactory', ['User', function(User) {
```

Comment:
Everything below this point looked like whitespace changes if I read it correctly.